rest-server/server/src/main/resources/reference.conf

```diff
@@ -22,3 +22,11 @@ auth {
   client-id = "harness"
   client-secret = "mwjmo4fwC9qFW081gm5vwsmAC4Z2747XEU8eeV6dzUzjW"
 }
+
+elasticsearch {
+  uri = "http://localhost:9200"
```

Comment:
URI should be in harness config, or env var, or provided here as a fallback. Like Mongo URI. This is for operation without Spark like queries etc. Anything needed for Spark should be generated from internal data or from engine.json `sparkConf`

The ES Rest host list is in the PIO+UR engine.json in the `sparkConf` since it is passed to Spark for `saveToEs`